### PR TITLE
CarPlay assets for a series of screens.

### PR DESCRIPTION
**Pull request by vincethecoder**
Assets provided to style specific screens shown below:
7-locate-day
7-locate-night
1-pre-guidance-day
1-pre-guidance-night
2-close-day
2-close-night
3-favorite-day
3-favorite-night
4-simulate-day
4-simulate-night
6-mute-day
6-mute-night
5-dont-simulate-day
5-dont-simulate-night

/cc @akitchen @designevokes

### CODE REVIEW

File: Examples/Swift/AppDelegate+CarPlay.swift [hunk 1]
```diff
@@ -52,11 +52,13 @@ extension AppDelegate: CarPlayManagerDelegate {
             return nil
         }
         
+        let disableSimulateText = "Don’t Simulate"
+        let enableSimulateText =  "        Simulate"
```
Comment (akitchen) [inline Examples/Swift/AppDelegate+CarPlay.swift]: hmm, I wish we had something better here

Comment (akitchen) [reply Examples/Swift/AppDelegate+CarPlay.swift]: How about "Enable Simulation"/"Disable Simulation"

It's verbose (perhaps too long?), but it's clear, and this button doesn't need to share real estate (for now)

Comment (vincethecoder) [reply Examples/Swift/AppDelegate+CarPlay.swift]: @akitchen Sure. I will make those changes.

**Comment by vincethecoder** [comment]
@akitchen updates shown below:

simulator screen shot - iphone 8 - 2018-09-15 at 17 40 27
simulator screen shot - iphone 8 - 2018-09-15 at 17 40 30

**Comment by akitchen** [comment]
👍 let's see how far we get with this button text. for now I think it provides the most clarity.

**Pull request merged**